### ISSUE TITLE
Scan In Stock Items can only do one at a time

### ISSUE THREAD

**Issue by sourpatchdyke** [inventree/InvenTree#8127]
### Please verify that this bug has NOT been raised before.

- [x] I checked and didn't find a similar issue

### Describe the bug*

When I use "Scan In Stock Items" on a stock location, it only accepts the first barcode I scan. The second causes the dialog to close and only the first is actually scanned in.

### Steps to Reproduce

1. Open a Stock Location.
2. Open "Scan In Stock Items".
3. Scan two barcodes.

### Expected behaviour

The dialog stays open and continues to accept barcodes.

### Deployment Method

- [x] Docker
- [ ] Package
- [ ] Bare metal
- [ ] Other - added info in Steps to Reproduce

### Version Information

# Version Information:
InvenTree-Version: 0.17.0 dev
Django Version: 4.2.15
Commit Hash: d4e67c0
Commit Date: 2024-09-13

Database: postgresql
Debug-Mode: False
Deployed using Docker: True
Platform: Linux-6.1.0-25-amd64-x86_64-with
Installer: DOC

Active plugins: False

### Please verify if you can reproduce this bug on the demo site.

- [ ] I can reproduce this bug on the demo site.

**Comment by sourpatchdyke** [comment]
Not checking "I can reproduce this bug on the demo site" shouldn't be taken to suggest it worked properly there. I don't have any barcodes on hand representing items on the demo site so I didn't try.

**Comment by SchrodingersGat** [comment]
@sourpatchdyke can you please provide some screenshots of the dialogs so we know exactly which part of the interface you are talking about?

**Comment by sourpatchdyke** [comment]
Yes, no problem. Here it is with one item scanned in. Any more and it just closes.

![Image](https://github.com/user-attachments/assets/5cd379e0-d41a-4e00-8646-91985514e906)

**Comment by SchrodingersGat** [comment]
Ok, thanks. We'll need to look into it. You are also very welcome to submit a fix for this if you can work it out yourself. Otherwise, it will be addressed when we have some developer time

**Comment by sourpatchdyke** [comment]
Cool. I have no idea how to fix this so I'll let y'all deal with it.

**Comment by matmair** [comment]
@SchrodingersGat this seems to regard CUI - should we close it out with the removal of CUI in #8384?

**Comment by SchrodingersGat** [comment]
@matmair yes, that makes sense
